lib/bolt/outputter/human.rb

```diff
@@ -452,29 +452,77 @@ def print_targets(target_list, inventoryfile)
         targets += target_list[:inventory].map { |target| [target.name, nil] }
         targets += target_list[:adhoc].map { |target| [target.name, adhoc] }
 
-        if targets.any?
-          @stream.puts format_table(targets, 0, 2)
-          @stream.puts
-        end
+        info = +''
+
+        # Add target list
+        info << colorize(:cyan, "Targets\n")
+        info << if targets.any?
+                  format_table(targets, 2, 2).to_s
+                else
+                  indent(2, 'No targets')
+                end
+        info << "\n\n"
+
+        @stream.puts info
+
+        print_inventory_summary(
+          target_list[:inventory].count,
+          target_list[:adhoc].count,
+          inventoryfile
+        )
+      end
+
+      def print_target_info(target_list, inventoryfile)
+        adhoc_targets     = target_list[:adhoc].map(&:safe_name).to_set
```

Comment:
Hm, admittedly I think this is kind of moot since the non-safe-name will get printed right below, unless the idea is to be more readable?
```
lucy ~/githubs/bolt ((f4d3f755...)) $ be bolt inventory show -t all --detail
bolt@localhost:20024
  name: bolt:bolt@localhost:20024
  uri: bolt:bolt@localhost:20024
```

Reply:
:chefs-kiss: Yeah, I think that's what users will be expecting to look for. Can you also update the commit message, just we don't also confuse our future selves? Then this is GtG!

Reply:
done